.github/scripts/deploy.sh

```
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Define the project root relative to the script location
+SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
+PROJECT_ROOT=$(dirname $(dirname "$SCRIPT_DIR")) # Go two levels up from script dir
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)